code/game/objects/structures/vehicles/vehicle.dm

```diff
@@ -222,6 +217,12 @@
 /obj/structure/bed/chair/vehicle/proc/can_buckle(mob/M, mob/user)
 	if(M != user || !ishuman(user) || !Adjacent(user) || user.restrained() || user.lying || user.stat || user.locked_to || destroyed || occupant)
 		return 0
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.mind && H.mind.special_role == "highlander")
+			if(user == M)
+				to_chat(user, "<span class='warning'>A true highlander has no need for a mount!</span>")
+			return
```

Comment:
Yes, but when we're putting return values like this its for clarity to show something is relying on those return values